backend/api/admin/src/main/java/co/airy/core/api/admin/sources/google/GoogleSource.java

```diff
@@ -0,0 +1,30 @@
+package co.airy.core.api.admin.sources.google;
+
+import co.airy.core.api.admin.Source;
+import co.airy.core.api.admin.SourceApiException;
+import co.airy.core.api.admin.dto.ChannelMetadata;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class GoogleSource implements Source {
+    @Override
+    public String getIdentifier() {
+        return "google";
+    }
+
+    @Override
+    public List<ChannelMetadata> getAvailableChannels(String token) throws SourceApiException {
+        return List.of();
```

Comment:
See #223: The flow is very specific and does not really map to the way Google channels are organized (hierarchically under brands). So until we get product input I would leave this open.